src/sourmash/index.py

```diff
@@ -201,7 +201,7 @@ def search_abund(self, query, *, threshold=None, **kwargs):
                 raise TypeError("'search_abund' requires subject signatures with abundance information")
             score = query.similarity(subj)
             if score >= threshold:
-                matches.append(IndexSearchResult(score, subj, self.location))
```

Comment:
pretty sure this was a bug, @ctb, but pointing out so you can double check. New test (test_search_ani_empty_abund) in test_sourmash.py revealed it -- the `filename` entry in the results csv was empty (+ is correct with this change).

Reply:
agreed, and nice catch! I might try to swipe this fix and combine it with a fix for https://github.com/sourmash-bio/sourmash/issues/1804 if that's ok by you?

Reply:
Sure!

Reply:
redundantly fixed in https://github.com/sourmash-bio/sourmash/pull/1820; resolving this conversation even though that hasn't been merged yet :)